src/main/java/com/ericsson/ei/handlers/ExtractionHandler.java

```diff
@@ -64,17 +64,18 @@ public void runExtraction(RulesObject rulesObject, String id, String event, Stri
             JsonNode aggregatedJsonObject = mapper.readTree(aggregatedDbObject);
             runExtraction(rulesObject, id, event, aggregatedJsonObject);
         } catch (Exception e) {
-            log.info(e.getMessage(),e);
+            LOGGER.info("Failed with extraction.", e);
         }
     }
 
     public void runExtraction(RulesObject rulesObject, String mergeId, String event, JsonNode aggregatedDbObject) {
         JsonNode extractedContent = extractContent(rulesObject, event);
 
         if(aggregatedDbObject != null) {
-            log.debug("ExtractionHandler: Merging Aggregated Object:\n" + aggregatedDbObject.toString() +
-            		"\nwith extracted content:\n" + extractedContent.toString() +
-            		"\nfrom event:\n" + event);
+            LOGGER.debug("ExtractionHandler: Merging Aggregated Object:\n{}" +
```

Comment:
Not sure but please look up if not + should be the start of a new line.

Reply:
Done.